client/components/map/mapToolTip.jsx

```diff
@@ -1,12 +1,15 @@
 import React from 'react'
 import { Popup } from 'react-leaflet'
-const Tooltip = ({distance, trailName, ...props}) => (
+const Tooltip = ({distance, trailName, isActive, ...props}) => (
   <Popup {...props}>
     <span className='trail__tooltip'>
       <h5>
         { trailName || 'No name given' }
       </h5>
-      <p>Estimated Distance: {guessLength(props.popupContainer, distance.addLength)} Miles
+      <p>Estimated Distance: {isActive.b 
```

Comment:
My previous comment got shoved into the dated commit. It is that way to currently render all results on load, per a conversation with malachi about design...